migrate/live-migration/index.md

```diff
@@ -9,10 +9,9 @@ import SourceTargetNote from "versionContent/_partials/_migrate_source_target_no
 
 # Live migration
 
-Live migration is a migration strategy to move a large amount of data
-(100&nbsp;GB-10&nbsp;TB+) with low downtime (on the order of few minutes). It requires
-more steps to execute than a migration with downtime using [pg_dump/restore][pg-dump-and-restore],
-but supports more use-cases and has less requirements than the [dual-write and backfill] method.
+Live migration is a strategy used to move a large amount of data (100&nbsp;GB-10&nbsp;TB+) with minimal downtime (typically a few minutes). It involves copying existing data from the source to the target and supports change data capture to stream ongoing changes from the source during the migration process.
```

Comment:
```suggestion
Live migration is a strategy to move a large amount of data
(100&nbsp;GB-10&nbsp;TB+) with minimal downtime (typically a few minutes). It
achieves low downtime by simultaneously 1) copying existing data from the
source database to the target database and 2) recording ongoing changes which
are made on the source. When the initial data copy completes, it continuously
applies the recorded transactions to the target database until the target
database is fully caught up with the source database. At this point the
application's database connection is switched to the target database (which may
result in a short downtime), and the migration is complete.
```

Reply:
I think `1)` & `2)` is not normal in docs. 

> When the initial data copy completes, it continuously
applies the recorded transactions

When the initial data copy completes, live migration applies the recorded transactions ...